cmd/ocm-backplane/console/console.go

```diff
@@ -675,6 +934,30 @@ func isRunningHigherThan411() bool {
 	return false
 }
 
+// isRunningHigherThan413 checks the running cluster is higher than 413
+func isRunningHigherThan413() bool {
+	currentClusterInfo, err := utils.DefaultClusterUtils.GetBackplaneClusterFromConfig()
+	if err != nil {
+		return false
+	}
+	currentCluster, err := ocm.DefaultOCMInterface.GetClusterInfoByID(currentClusterInfo.ClusterID)
+
+	if err != nil {
+		return false
+	}
+	clusterVersion := currentCluster.OpenshiftVersion()
+	if clusterVersion != "" {
+		version, err := semver.NewVersion(clusterVersion)
+		if err != nil {
+			return false
+		}
+		if version.Minor() >= 14 {
```

Comment:
can you define  `4.13` version as constant and rename the function `isClusterVersionHigherThan( expectedVersion ) bool`